src/uikit.cs

```diff
@@ -3284,14 +3284,14 @@ interface UIContextMenuInteractionDelegate {
 		[return: NullAllowed]
 		UITargetedPreview GetPreviewForDismissingMenu (UIContextMenuInteraction interaction, UIContextMenuConfiguration configuration);
 
-		[Export ("contextMenuInteraction:willCommitWithAnimator:")]
-		void WillCommit (UIContextMenuInteraction interaction, IUIContextMenuInteractionCommitAnimating animator);
+		[Export ("contextMenuInteraction:willPerformPreviewActionForMenuWithConfiguration:animator:")]
+		void WillPerformPreviewActionForMenu (UIContextMenuInteraction interaction, UIContextMenuConfiguration configuration, IUIContextMenuInteractionCommitAnimating animator);
 
-		[Export ("contextMenuInteractionWillPresent:")]
-		void WillPresent (UIContextMenuInteraction interaction);
+		[Export ("contextMenuInteraction:willDisplayMenuForConfiguration:animator:")]
+		void WillDisplayMenu (UIContextMenuInteraction interaction, UIContextMenuConfiguration configuration, [NullAllowed] IUIContextMenuInteractionAnimating animator);
 
-		[Export ("contextMenuInteractionDidEnd:")]
-		void DidEnd (UIContextMenuInteraction interaction);
+		[Export ("contextMenuInteraction:willEndForConfiguration:animator:")]
+		void WillEndForConfiguration (UIContextMenuInteraction interaction, UIContextMenuConfiguration configuration, [NullAllowed] IUIContextMenuInteractionAnimating animator);
```

Comment:
`ForConfiguration` seems redundant with the parameter. Swift uses overloads for all `contextMenuInteraction` so it's not really a good substitute for the name.

Reply:
And I'm not quite sure about the other `ForXXX`

Reply:
Yeah I think plain `WillEnd` is ok

```suggestion
		void WillEnd (UIContextMenuInteraction interaction, UIContextMenuConfiguration configuration, [NullAllowed] IUIContextMenuInteractionAnimating animator);
```